### DISCUSSION TITLE
Is it possible to run the checker on a cluster of machines?

### DISCUSSION THREAD

**Discussion by ekexium** [p-org/P#435] [Q&A] [answered]
If I have a complex model that may take too much time to check, is it possible to run it on a cluster?
Seems the document implies this, but I didn't find how.
Did I miss anything?

**Comment by ankushdesai** [comment, marked as answer]
You would have to setup your own cluster and then run the P checker on it. We dont have an open source service.

**Reply by ekexium** [reply]
Does that mean there is no way to make schedules explored by each machine not overlap? Unless I write and assign different tests for them?

**Reply by ankushdesai** [reply]
You are right! But in most cases the strategy would be to run locally for 100K iteration remove all the shallow bugs and then run on a distributed cluster with different search strategies. Please let me know when you get to the later stage and I can guide you on how to setup the runs on cluster!

Thanks!!

**Reply by ekexium** [reply]
Cool. Thank you!
